src/components/Markdown/Markdown.tsx

```diff
@@ -17,7 +17,8 @@ export interface MarkdownProps {
   markdownCSSOverrides?: Record<string, React.CSSProperties | string | number>;
   forceMultiLine?: boolean;
   multiLineCodePropOverrides?: Partial<MultiLineCodeProps>;
-}
+} 
+
```

Comment:
try to not unnecessarily add new lines in PRs accidentally. It's a good practice to review your PRs right before or after submitting them. I like to use github desktop for this

Reply:
Don't forget about t his change